mm2src/coins/z_coin/storage/z_change_notes/sqlite.rs

```diff
@@ -0,0 +1,133 @@
+use super::{ChangeNote, ChangeNoteStorage, ChangeNoteStorageError};
+
+use db_common::async_sql_conn::AsyncConnError;
+use db_common::sqlite::rusqlite::params;
+use db_common::sqlite::{run_optimization_pragmas, validate_table_name};
+use itertools::Itertools;
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::prelude::*;
+
+fn table_name(addr: &str) -> String { format!("{addr}_change_notes_cache") }
```

Comment:
Here we segregate change notes by addresses.
But the spendable balance obtained in get_balance fn is queried by the AccountId. Should this not be done identically?

Also, why not just have a column with the address instead of adding it to the table name?